### DISCUSSION TITLE
Missing search in repositories

### DISCUSSION THREAD

**Discussion by landsman** [community/community#53760] [Mobile]
### Select Topic Area

Product Feedback

### Body

In iOS mobile app is missing search in my repositories.

Steps to reproduce:
1. open app (home tab)
2. in section "My work" click to Repositories
3. I'm able to filter them only by type 😢

Search is present in other items in this section. I can search in my repositories from Profile, but I would expect the same from "My work" tab.

**Comment by taki-on** [comment]
Thank you for your suggestion! Search and filtering in my repositories are currently in our backlog and we want to work on them soon. Please stay tuned!

**Comment by masifmehmoodsami99** [comment]
Thnkxxx for you

**Reply by masifmehmoodsami99** [reply]
- [ ]
